common/Log.hpp

```diff
@@ -311,69 +311,76 @@ static constexpr std::size_t skipPathPrefix(const char (&s)[N], std::size_t n =
     do                                                                                             \
     {                                                                                              \
         auto& log_ = Log::logger();                                                                \
+        auto& name_ = log_.name();                                                                 \
```

Comment:
So - this is extra cost in all the hot paths - if we do this it really should be in the (very unusual) if () condition where it is actually used.